walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +272,155 @@ public void notEnoughRoom() {
+    // We should see the two options that fit the mandatory attendees, ignoring the optional atendee,
+    // because there is no slots available for both mandatory and optional attendees.
+    //
+    // Events of mandetory attendees  :        |--A--|     |--B--|
+    // Events of optional attendee    :  |-------------C---------------|
+    // Day                            :  |-----------------------------|
+    // Options to return              :  |--1--|     |--2--|     |--3--|
+
+    Collection<Event> events = Arrays.asList(
+        new Event("Event 1", TimeRange.fromStartDuration(TIME_0800AM, DURATION_30_MINUTES),
+            Arrays.asList(PERSON_A)),
+        new Event("Event 2", TimeRange.fromStartDuration(TIME_0900AM, DURATION_30_MINUTES),
+            Arrays.asList(PERSON_B)),
+        new Event("Event 3", TimeRange.WHOLE_DAY, Arrays.asList(PERSON_C)));
+
+    MeetingRequest request =
+        new MeetingRequest(Arrays.asList(PERSON_A, PERSON_B), DURATION_30_MINUTES);
+        request.addOptionalAttendee(PERSON_C);
+
+    Collection<TimeRange> actual = query.query(events, request);
+    Collection<TimeRange> expected =
+        Arrays.asList(TimeRange.fromStartEnd(TimeRange.START_OF_DAY, TIME_0800AM, false),
+            TimeRange.fromStartEnd(TIME_0830AM, TIME_0900AM, false),
+            TimeRange.fromStartEnd(TIME_0930AM, TimeRange.END_OF_DAY, true));
+
+    Assert.assertEquals(expected, actual);
+  }
+
+
+  @Test
+  public void availableTimeSlotsWithOptionalAtendee() {
+    // Optional atendee has an event so that one time slot that fits the mandatory attendees is not suitable for him,
+    // and two time slot fits bot mandatory and optional attendees. 
+    // Only those two time slots should be returned.
+    //
+    // Events of mandetory attendees  :        |--A--|     |--B--|
+    // Events of optional attendee    :              |--C--|
+    // Day                            :  |-----------------------------|
+    // Options to return              :  |--1--|                 |--2--|
+
+    Collection<Event> events = Arrays.asList(
+        new Event("Event 1", TimeRange.fromStartDuration(TIME_0800AM, DURATION_30_MINUTES),
+            Arrays.asList(PERSON_A)),
+        new Event("Event 2", TimeRange.fromStartDuration(TIME_0900AM, DURATION_30_MINUTES),
+            Arrays.asList(PERSON_B)),
+        new Event("Event 3", TimeRange.fromStartDuration(TIME_0830AM, DURATION_30_MINUTES),
+            Arrays.asList(PERSON_C)));
+
+    MeetingRequest request = new MeetingRequest(Arrays.asList(PERSON_A, PERSON_B), DURATION_30_MINUTES);
+    request.addOptionalAttendee(PERSON_C);
+
+    Collection<TimeRange> actual = query.query(events, request);
+    Collection<TimeRange> expected =
+        Arrays.asList(TimeRange.fromStartEnd(TimeRange.START_OF_DAY, TIME_0800AM, false),
+            TimeRange.fromStartEnd(TIME_0930AM, TimeRange.END_OF_DAY, true));
+    Assert.assertEquals(expected, actual);
```

Comment:
Conventionally the order should be assertEquals(actual, expected).
In addition, notice you can use a static import on assertEquals:
import static org.junit.Assert.assertEquals;

and then use assertEquals without the Assert. prefix.